source/setup.py

```diff
@@ -6,19 +6,23 @@
 #See the file COPYING for more details.
 
 import os
+import sys
 import copy
 import gettext
 gettext.install("nvda")
 from setuptools import setup
 import py2exe as py2exeModule
 from glob import glob
 import fnmatch
+# versionInfo names must be imported after Gettext
+from versionInfo import formatBuildVersionString, name, version, publisher  # noqa: E402

```

Comment:
1. Please add the name of the suppressed lint error for E402
1. Please change this to the one import per line style, you can group with parentheses:
```
from versionInfo import ( 
	formatBuildVersionString,
	etc,
)
```